selfdrive/ui/qt/onroad.cc

```diff
@@ -1,26 +1,27 @@
 #include "selfdrive/ui/qt/onroad.h"
 
-#include <iostream>
 #include <QDebug>
 
-#include "selfdrive/common/swaglog.h"
 #include "selfdrive/common/timing.h"
 #include "selfdrive/ui/paint.h"
 #include "selfdrive/ui/qt/util.h"
 #ifdef ENABLE_MAPS
 #include "selfdrive/ui/qt/maps/map.h"
 #endif
 
+VisionStreamType getVisionStreamType() {
+  bool wide_cam = Hardware::TICI() && Params().getBool("EnableWideCamera");
+  return wide_cam ? VISION_STREAM_RGB_WIDE : VISION_STREAM_RGB_BACK;
```

Comment:
we should rename the vision streams using the new road/wideRoad/driver convention